### PR TITLE
adding new cols to participation dict

### PR DESCRIPTION
**Pull request by john-b-edwards**
Remind me again -- do I have to run some kind of pkgdown command or is that already covered?

### CODE REVIEW

**Comment by tanho63** [comment]
Nothing with pkgdown provided you ran build_data_dictionaries

**Review by tanho63 — COMMENTED** [review]
Can you bump the DESCRIPTION version and add a news bullet?

**Comment by john-b-edwards** [comment]
Yep yep did that

File: data-raw/dictionary_participation.csv [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+offense_personnel,character,"Number of running backs, tight ends, and wide receivers on the field for the play. If there are more than the standard 5 offensive linemen and 1 quarterback, they will be listed here as well."
+defenders_in_box,integer,Number of defensive players lined up in the box at the snap.
+defense_personnel,character,"Number of defensive linemen, linebackers, and defensive backs on the field for the play. "
+number_of_pass_rushers,integer,Number of defensive player who rushed the passer.
+players_on_play,character,"A list of every player on the field for the play, by gsis_it_id"
+offense_players,character,"A list of every offensive player on the field for the play, by gsis_id"
+defense_players,character,"A list of every defensive player on the field for the play, by gsis_id"
+n_offense,integer,Number of offensive players on the field for the play
+n_defense,integer,Number of defensive players on the field for the play
+ngs_air_yards,double,Distance (in yards) that the ball traveled in the air on a given passing play as tracked by NGS
```
Comment (tanho63) [inline data-raw/dictionary_participation.csv:16]: Oh is this as-the-crow-flies (ie diagonals) or just “yardage from QB launch point to WR catch point”? I wonder if we should call it air distance to reduce confusion with nflverse air yards. Also I realize this probably should have been resolved in the other PR…dangit.

**Review by tanho63 — APPROVED** [review]
LGTM ty

**Pull request merged**